### ISSUE TITLE
Add ability to filter on specific devices (or any tag)

### ISSUE THREAD

**Issue by adamreisnz** [getsentry/relay#956]
The feature to quickly filter out legacy browsers is great, however, I have some other old devices that are causing fluke bugs and I'd like to filter them out. To my surprise, I couldn't find out anywhere how to filter out specific errors except by using `ignoreErrors`, which I think only compares the error message body.

I would like for example, to be able to ignore all errors with tag `device:family` and value `Alcatel One Touch 7041X`.

Is it possible to implement such manual error filters on inbound data that look at specific tags?

If this is too hard to implement, could you add Android 4 as a legacy device?

**Comment by jan-auer** [comment]
There are no plans to add further capabilities to the existing inbound data filters. Dynamic Sampling is in very early development, so for the time being this is a wontfix. Once dynamic sampling is ready, we will communicate this across the product.

As @dashed mentions above, you can filter for specific devices in the Issue Stream, Discover, and Metric Alerts. This should allow to customize views and hide unwanted information.

**Comment by bengentil** [comment]
Our use-case is filtering hosts/clusters during maintenance or hosts spamming with known issues to avoid the cleanup of many unwanted events/issues (+VACUUM the database when an host is spaming millions of events).
Some hosts are NATed, thus the filtering on the relay.

I understand I can filter in the Issue stream, but it could be nice if the relay could prevent unwanted events to be created.
